pkg/kubectl/cmd/convert.go

```diff
@@ -152,7 +156,6 @@ func (o *ConvertOptions) Complete(f cmdutil.Factory, out io.Writer, cmd *cobra.C
 
 	// build the printer
 	o.out = out
-	outputFormat := cmdutil.GetFlagString(cmd, "output")
 	templateFile := cmdutil.GetFlagString(cmd, "template")
 	if len(outputFormat) == 0 {
```

Comment:
Does this check need to be adjusted here?
`len(outputFormat) == 0` is now being checked twice in this function.